communitytoolkit/TOC.yml

```
@@ -1,2 +1,8 @@
-- name: Index
-  href: index.md
\ No newline at end of file
+- name: Community Toolkits Documentation
+  href: index.md
+- name: .NET
+  href: https://docs.microsoft.com/en-gb/windows/communitytoolkit/
+- name: Windows
+  href: https://docs.microsoft.com/en-gb/windows/communitytoolkit/
+- name: MAUI
+  href: maui/index.md
```

Let's add `Community Toolkit` to ensure that users don't think they'll be taken to the .NET Docs / Windows Docs / MAUI Docs.

Let's also remove the culture (in this case en-gb) from Microsoft links. The Microsoft website will automatically add the culture, localizing the website based on the viewer's locale.

```suggestion
- name: .NET Community Toolkit
  href: https://docs.microsoft.com/windows/communitytoolkit/
- name: Windows Community Toolkit
  href: https://docs.microsoft.com/windows/communitytoolkit/
- name: MAUI Community Toolkit
  href: maui/index.md
```